src/components/UI/XButton/XButton.module.css

```diff
@@ -20,9 +20,11 @@
 }
 
 .DisplayInterests {
-    position: absolute;
-    right: 2px;
-    top: 2px;
-    font-size: 20px;
-    color: #6831de;
+    font-size: 16px;
```

Comment:
In your mockup. I didn't see displayInterests. Why did you change css here?

Reply:
Hi Arnas, inside interest section, there are three Xbuttons that contains tags for each interest. Aren't they suppose to be the DisplayInterests?

Reply:
Yes, you right on that one.